### ISSUE TITLE
Installation of v1.4.0 fails with latest version of NPM

### ISSUE THREAD

**Issue by Extend-Apps** [oracle/netsuite-suitecloud-sdk#544]
### Your environment
**OS** : MacOs Monterey v12.2.1
**Node.js version** : 16.13.0
**NPM.js version** : 8.5.0
**suitecloud-cli version** : 1.3.2 (Attempting to update to v1.4.0)
**Terminal/CMD tool** : /bin/zsh

### Describe the bug
Attempting to update to suitecloud v1.4.0

### To Reproduce
1. Run _**npm -g install @oracle/suitecloud-cli@1.4.0**_ or (**_npm install -g --supressSuiteCloudSDKLicensePrompt @oracle/suitecloud-cli_**)

### Actual Behavior
Skips past user prompt to accept license agreement, which results in `Installation aborted by user.` error

### Expected Behavior
To allow prompt and ability to say 'yes' (or respect the `supressSuiteCloudSDKLicensePrompt` toggle and auto-accept.

### Additional context
Worked with @alii-gator in a screen share.  We feel its an issue with the NPM version.

**Comment by wmadruga** [comment]
Same issue on linux.
Kernel 5.16.9-200
node 16.13.2
npm 8.1.2

Is there a workaround? I didn't want to revert my whole setup to use sdfcli again...

**Comment by Extend-Apps** [comment]
No workaround that I know of.  I just reverted to v1.3.2

**Comment by wmadruga** [comment]
I couldn't find any either and I was hoping I'd miss something :D Thanks!

**Comment by Extend-Apps** [comment]
[2022-03-02T14_17_23_327Z-debug-0.log](https://github.com/oracle/netsuite-suitecloud-sdk/files/8170120/2022-03-02T14_17_23_327Z-debug-0.log)

**Comment by 13uan** [comment]
Same issue on Windows 10.

Windows 10 Pro
node v16.13.2
npm 8.1.2

[2022-03-01T23_17_33_234Z-debug.log](https://github.com/oracle/netsuite-suitecloud-sdk/files/8170564/2022-03-01T23_17_33_234Z-debug.log)

**Comment by arook** [comment]
same issue.
anyone resolved?

**Comment by Extend-Apps** [comment]
No, I reverted to 1.3.2 and will wait for the next update.

**Comment by Syedali92** [comment]
@arook If you switch to node 14 (which will have npm 6.x), This will solve the issue.

**Comment by alii-gator** [comment]
@arook You can solve this by using node 14 LTS (npm 6.x)

**Comment by Extend-Apps** [comment]
@alii-gator , right.  All due respect, those supported versions are **2** major versions behind.  The current node is 16.14 and the current npm is 8.5.3.  Curious as to why suitecloud chooses to be so far behind?

**Comment by alii-gator** [comment]
We understand your concern. Node 14 is also LTS.
In Node 16 there are some changes made which are not backwards compatible. So we are making changes on our end to support node 16 officially. And there are some reported issues with npm 8 which affect some of our users. Rest assured we are making changes on our end to resolve this Asap.

**Comment by alii-gator** [comment]
@Extend-Apps Can you try with the latest version `1.5.0`
This should be fixed. Feel free to open this issue if you have any problems.

**Comment by alii-gator** [comment]
@arook @wmadruga please check with `1.5.0`
